var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -62,7 +63,7 @@ class Trilinos(CMakePackage, CudaPackage):
     # Build options
     variant('complex', default=False, description='Enable complex numbers in Trilinos')
     variant('cuda_rdc', default=False, description='turn on RDC for CUDA build')
-    variant('cxxstd', default='11', values=['11', '14', '17'], multi=False)
+    variant('cxxstd', default='14', values=['11', '14', '17'], multi=False)
```

Comment:
@sethrj I'm not sure if its possible to set a default of `11` for `13.0.1` and `14` for `13.2.0`

I'm seeing some build errors when `13.0.1` is built with `14`.  From xsdk POV I can always encode the corresponding cxxstd in xsdk/package.py [irrespective of this default]

Reply:
Yuck. Clingo can often change variants automatically so let's add conflicts for known broken variants and set the default to 14?

Reply:
The builds that failed  when using 14 with 13.0.1 [so far] are intel@19.0.3.199 on both my workstation and Cori.

Reply:
OK, do you know what version of `libstc++` (usually tied to a specific GCC) those were using?

Reply:
On workstation: (icpc-19.0.3.199/g++-7.4.0)
```
 libstdc++.so.6 => /nfs/apps/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/gcc-7.4.0-4kdemuwlzds2ofpkkz7yytgi7kyojuvz/lib64/libstdc++.so.6
```
On cori (icpc-19.0.3.199/g++-7.5.0)
```
 libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00002aaaaacd3000)
```

Reply:
If using `13.0.1` as `preferred` we should remove this `cxxstd` change?